corehq/ex-submodules/pillowtop/processors/form.py

```diff
@@ -34,7 +34,7 @@ def process_change(self, pillow_instance, change):
             # the same effect, an app having has_submissions set to True.
             mark_has_submission(domain, build_id)
 
-        user_id = doc.get('user_id')
+        user_id = doc.get('form', {}).get('meta', {}).get('userID')
```

Comment:
ah this might be why - https://github.com/dimagi/commcare-hq/blob/master/corehq/form_processor/serializers.py#L44 (thought didn't look closely). yeah the other method only works on sql, forgot we didnt' store the user_id directly on the model for couch